plaid/src/main.rs

```diff
@@ -83,15 +82,15 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
     let executor = Arc::new(executor);
 
     info!("Configured Webhook Servers");
-    let webhook_servers: Vec<Box<Pin<Box<dyn Future<Output = ()>>>>> = config
+    let webhook_servers: Vec<Box<Pin<Box<_>>>> = config
         .webhooks
         .into_iter()
         .map(|(server_name, config)| {
             let log_sender = log_sender.clone();
             let server_address: SocketAddr = config
                 .listen_address
                 .parse()
-                .expect("{server_name} had an invalid address");
+                .expect("A server had an invalid address");
```

Comment:
Just curious. Is there a reason for this?

Reply:
Server name never worked